qt/mainwindow.cpp

```diff
@@ -131,7 +131,7 @@ MainWindow::MainWindow(Framework & framework, bool apiOpenGLES3,
   CreateNavigationBar();
   CreateSearchBarAndPanel();
 
-  QString caption = qAppName();
```

Comment:
What is returned here?

Reply:
What's in [applicationName](https://doc.qt.io/qt-5/qcoreapplication.html#applicationName-prop) ? For some reason, I don't see any docs about qAppName.

It would be great to take it somehow from CMake, to distinguish Designer build vs normal build.